metadata-models/src/main/pegasus/com/linkedin/common/InputField.pdl

```diff
@@ -0,0 +1,22 @@
+namespace com.linkedin.common
+
+import com.linkedin.schema.SchemaField
+
+/**
+ * Information about a field a chart or dashboard references
+ */
+record InputField {
+  /**
+   * Copied version of the referenced schema field object for indexing purposes
+   */
+  schemaField: optional SchemaField
```

Comment:
move this field below the urn field, just from a readability perspective, the urn is the primary thing and this is a maybe filled out thing for indexing.

Reply:
done